kimchi/src/circuits/constraints.rs

```diff
@@ -331,17 +331,25 @@ impl<F: FftField + SquareRootField> LookupConstraintSystem<F> {
             Some(lookup_used) => {
                 let d1_size = domain.d1.size();
 
+                let (lookup_selectors, gate_lookup_tables) =
+                    lookup_info.selector_polynomials_and_tables(domain, gates);
+
                 // get the last entry in each column of each table
-                let dummy_lookup_values: Vec<Vec<F>> = lookup_tables
+                let dummy_lookup_values: Vec<Vec<F>> = gate_lookup_tables
                     .iter()
+                    .chain(lookup_tables.iter())
```

Comment:
So these are non-fixed lookup tables?

Reply:
in which case, I would suggest making that argument an `Option`, and also does that work currently or is that a step in the direction of making it work?

Reply:
IMO an option is redundant when we're passing a slice/vector, since `None` should be equivalent to the empty slice (and indeed `Vec::with_capacity(0)` is non-allocating, so it's a drop-in replacement).